lightning/src/ln/channel.rs

```diff
@@ -4451,6 +4480,13 @@ impl<Signer: Sign> Channel<Signer> {
 				htlc.state = OutboundHTLCState::AwaitingRemovedRemoteRevoke(fail_reason);
 			}
 		}
+		if let Some((feerate, update_state)) = self.pending_update_fee {
+			if update_state == FeeUpdateState::AwaitingRemoteRevokeToAnnounce {
```

Comment:
Looks like pending_update_fee docs just need updating, I'll do that in a followup - the point being that we now treat the update_fee state exactly the same way as we treat `pending_inbound_htlc` update state, because they are the same thing, basically, in terms of the ln state machine.